lib/install/management/store_files.go

```diff
@@ -350,8 +350,10 @@ func (d *Dispatcher) deleteVolumeStoreIfForced(conf *config.VirtualContainerHost
 			dsURL.Path = path.Join(dsURL.Path, constants.VolumesDir)
 		}
 
-		d.op.Debugf("Provided datastore URL: %q\nParsed volume store path: %q", url.Path, dsURL.Path)
-		d.op.Infof("Deleting volume store %q on Datastore %q at path %q", label, dsURL.Host, dsURL.Path)
+		d.op.Debugf("Provided datastore URL: %q", url.Path)
+		d.op.Debugf("Parsed volume store path: %q", dsURL.Path)
+		d.op.Infof("Deleting volume store %q on Datastore %q at path %q",
```

Comment:
Please don't be this aggressive about line wrapping - it's not necessary with modern monitors and environments and makes it much harder to quickly scan for logic flow.